docs/usage.md

```diff
@@ -284,18 +294,18 @@ Full path to Kraken2 database built from viral genome.
 
 Name for host genome as recognised by Kraken2 when using the `kraken2 build` command as configured on `genomes.config`. Default: 'viral'.
 
-### `--host_kraken2_db`
-
-Full path to Kraken2 database built from host genome.
-
-### `--host_kraken2_name`
+### `--kraken2_use_host_db`

```

Comment:
What do you think of an opposite `--kraken2_use_viral_db` flag, one of which defaults to true?

Or alternatively, `--kraken2_db : { 'host', 'viral' }`?

Or is this flag necessary at all, can we just check `host_kraken2_db` and `viral_kraken2_db`?

Reply:
Yes, I am not 100% happy with the implementation for Kraken. Will refine this all before the release 👍